Drupal 11 Upgrade Preparation

quicksketch opened the pull request:
Similar to the PR at https://github.com/MukurtuCMS/Mukurtu-CMS/pull/707, except this doesn't actually upgrade to Drupal 11; it only prepares for it.

- Upgrades contrib modules to latest versions that support Drupal 11
- Updates all Feature modules to state Drupal 11 compatibility.
- Removes/updates all deprecated custom PHP code.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
@quicksketch Only regression I've spotted so far is the DH content page view isn't displaying media assets/media carousel.

That might be Colorbox related?

![Screenshot 2025-02-28 at 8 53 16 AM](https://github.com/user-attachments/assets/a7aeb56b-d87c-4c68-b7ea-73befa1c6a3f)

![Screenshot 2025-02-28 at 8 53 25 AM](https://github.com/user-attachments/assets/0c711a1f-096d-4ba3-89dd-5dc7e5154609)

quicksketch commented:
Thanks @michael-wynne-wsu, I'm looking into it. Looks like it's caused by a JavaScript error on the page that is coming from Leaflet, due to the module upgrade. I'm checking how we can update our custom Leaflet code to be compatible with the newer version.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr712-x4ili6k6g35sxgtvr19b782v6gvaa5rt.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67bfd1118eeb0f9feb7310b9

quicksketch commented:
We updated our custom Leaflet code as part of https://github.com/MukurtuCMS/Mukurtu-CMS/issues/110.

I updated this PR with `main`, and if Leaflet still works well, this should be good to merge to continue prepping for D11.

michael-wynne-wsu commented:
@quicksketch Carousel and Leaflet both look good to me, feel free to merge!
